### ISSUE TITLE
Tee function?

### ISSUE THREAD

**Issue by hadley** [tidyverse/magrittr#135]
i.e.

```R
tee <- function(x, ...) x
```

Because it looks slightly nicer to my eye?  See e.g. https://github.com/hadley/testthat/issues/482

**Comment by hadley** [comment]
Also this just occurred to me:

```R
safetee <- function(x, ...) {
  try(list(...))
  x
}
```

**Comment by hadley** [comment]
The puns keep coming:

```R
finalitee <- function(x, ..., finally) {
  if (!missing(finally)) 
    on.exit(finally)

  list(...)
  x
}

**Comment by smbache** [comment]
Love the puns :)

Would these belong in magrittr, though, if we remove the aliases? They could also join them in the `nestpas` pkg, so magrittr is operators only?

**Comment by smbache** [comment]
Could we find useful functionality for a `fatalitee` too ;)

**Comment by hadley** [comment]
Yeah, might be better in nestpas, although these feel a bit more pipe related than the aliases.

Could also have a tee micropackage.

**Comment by smbache** [comment]
Maybe that could be called `fatalitee` :) for no apparent reason.. but the mk voice would play in many minds ...
